src/components/Files/FilesTab.tsx

```diff
@@ -126,6 +127,7 @@ export const FilesTab = (props: FilesTabProps) => {
   const fileUpload = useFileUpload({
     type: type,
     multiple: true,
+    CombineToPDF: isPdf,
```

Comment:
If you take this approach, what will reset the isPdf flag?

Also modify allowedExtensions depending on isPdf flag.